docs/reference/wheel_tags.rst

```diff
@@ -0,0 +1,60 @@
+wheel tags
+==========
+
+Usage
+-----
+
+::
+
+    wheel tagstags [-h] [--remove] [--python-tag TAG] [--abi-tag TAG] [--platform-tag TAG] [--build NUMBER] WHEEL [...]
+
+Description
+-----------
+
+Make a new wheel with given tags from and existing wheel. Any tags left
+unspecified will remain the same. Multiple tags are separated by a "." Starting
+with a "+" will append to the existing tags.  Starting with a "-" will remove a
+tag. The original file will remain unless ``--remove`` is given. The output
+filename(s) will be displayed on stdout for further processing.
+
+
+Options
+-------
+
+.. option:: --remove
+
+    Remove the original wheel, keeping only the retagged wheel.
+
+.. option:: --python-tag TAG
```

Comment:
Perhaps you'll want to suggest `--*-tag=TAG` syntax, as the value may start with `-`

Reply:
Oops, you are correct, thanks!

(Sadly, the output from `--help` from argparse will suggest the space)

Reply:
Fixed. Maybe we should include a warning in either the docs or the help output?

Reply:
Probably a good idea.